src/main/webapp/static/css/styles.css

```diff
@@ -128,9 +128,7 @@ footer.page-footer .footer-copyright {
 .btn.tokenButton {
     border-radius: 32px;
     background: linear-gradient(135deg, #b39ddb 0%, #9575cd 100%); /* deep-purple lighten-3 --> #9575cd deep-purple lighten-2 */
-}
-.nav-wrapper .btn.tokenButton {
-    background: linear-gradient(135deg, rgba(255,255,255, .4) 0%, rgba(255,255,255, .2) 100%);
```

Comment:
Please leave this code, so that the button can adapt itself to other colors.

<img width="878" alt="Screen Shot 2021-09-09 at 4 05 20 PM" src="https://user-images.githubusercontent.com/15718174/132682709-2807b295-cb64-48be-aaec-f70b8db9351d.png">

Reply:
Done!